tests/e2e/encrypted.sh

```diff
@@ -0,0 +1,179 @@
+#!/bin/bash
+#
+# Copyright Confidential Containers Contributors
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+script_dir="$(dirname "$(readlink -f "$0")")"
+source "${script_dir}/lib.sh"
+
+export key_file="image_key"
+export key_path="/default/image_key/nginx"
+export trustee_repo="https://github.com/confidential-containers/trustee.git"
+export kbs_namespace="coco-tenant"
+export aa_kbc="${aa_kbc:-cc_kbc}"
+export kbs_svc_name="kbs"
+export kbs_ingress_name="kbs"
+export runtimeclass="kata-qemu"
+export encrypted_image="ghcr.io/confidential/nginx:latest"
```

Comment:
```suggestion
export encrypted_image="ghcr.io/confidential-containers/nginx:encrypted"
```

Reply:
If we can pass in the repo via an envar and set it in the workflow that would be even better

Reply:
changes applied